taxonomy.simple.yaml

```diff
@@ -54,209 +43,154 @@
         - Parenting Education
         - Virtual Support
     - Relief for Caregivers
-    
+
   - Community Services:
     - Community Gardens
     - Computer or Internet Access
     - Community center
     - Enrichment Programs
     - Exercise & Fitness
-    - Helplines:
-      - Emergency Helplines
-      - Advice Helplines
+    - Libraries
+    - Helplines
     - Olim Support
     - Recreation
     - Religious Services:
       - Synagogues
       - Mosques
       - Churches
-      - Gamach
-      - Kollel
       - Altenative Religious Services
     - Social Club
     - Summer Camps
-  
+    - Assistance during mourning period
+
   - Education:
     - Help Find School
-    - Help Pay for School:
-      - Books
-      - Clothes for School
-      - Financial Aid & Loans
-      - Supplies for School
-      - Transportation for School
+    - Education Institutions:
+      - Nursery
+      - Kindergarten
+      - Primary School
+      - Middle School
+      - High School:
+        - High School Yeshiva
+        - Ulpana
+        - Youth Villages
+      - Further Education:
+        - Engineering
+        - Yeshiva and Kollel
+      - Academia
     - More Education:
       - Alternative Education
       - Civic Engagement
       - College Readiness
+      - Army Readiness
+      - National Service Readiness
       - English as a Second Language (ESL)
       - Enrichment Programs
       - Environmental Education
-      - Financial Education:
-        - Credit Counseling
-        - Homebuyer Education
+      - Financial Education
       - Foreign Languages
       - GED/High-School Equivalency
-      - Health Education:
-        - Family Planning
-        - Nutrition Education
-        - Parenting Education
-        - Safety Education
-        - Sex Education
-      - Prep - Mechina
-      - Special Education
-      - Summer Camps
+      - Prep - Mechina:
+        - Army Prep
+        - University Prep
       - Tutoring
       - Youth Development
       - Youth Movement
     - Leadership development
-    - Preschool
     - Screening & Exams:
-      - English as a Second Language (ESL)
       - GED/High-School Equivalency
+    - English as a Second Language (ESL)
```

Comment:
Duplicate

Reply:
Not a duplicated actually but needed to be deleted non the less. Good catch.